### DISCUSSION TITLE
List of toolboxes which uses Casadi

### DISCUSSION THREAD

**Discussion by brunoml5** [casadi/casadi#3157] [General]
Hi!

Is there a list of toolboxes which uses Casadi? Some time ago I saw a list somewhere but I don't remember. Maybe it would be a nice addition to the Wiki pages.

I have the following links saved in my browser:

- [MPCTools](https://bitbucket.org/rawlings-group/mpc-tools-casadi/src/master/): MPC and MHE, Rawlings' group
- [paresto](https://github.com/rawlings-group/paresto): Parameter estimation, Rawlings' group
- [acados](https://github.com/acados/acados): Real-time MPC, Diehl's group
- [TuneMPC](https://github.com/jdeschut/tunempc/): economic tuning of MPC, uses acados and casadi, from Zanon and Diehl
- [do-MPC](https://github.com/do-mpc/do-mpc): Robust multi-stage MPC, MHE, Lucia's group
- [rockit](https://gitlab.kuleuven.be/meco-software/rockit): Gillis' group
- [UKF-SNMPC](https://github.com/Eric-Bradford/UKF-SNMPC): Stochastic MPC with UKF (Bradford)
- [SDD-GP-MPC](https://github.com/Eric-Bradford/SDD-GP-MPC): Gaussian processes and MPC (Bradford)
- [casiopeia](https://github.com/adbuerger/casiopeia): Parameter estimation and optimal experiment design
- [NLoed](https://github.com/NateBraniff/NLoed): optimal experiment design
- [ml-casadi](https://github.com/TUM-AAS/ml-casadi): PyTorch + Casadi

**Comment by jgillis** [comment]
Some are listed at the bottom of our homepage casadi.org

Here is also a list: https://github.com/meco-group/dynamic_optimization_inventory/blob/main/list.csv

Other:
* pybamm

**Reply by brunoml5** [reply]
> Some are listed at the bottom of our homepage casadi.org
> 
I guess I forgot about this. Sorry.

> Here is also a list: https://github.com/meco-group/dynamic_optimization_inventory/blob/main/list.csv
> 

That is the list that I was talking about. Awesome.

**Comment by jgillis** [comment]
Gathering an official list here: https://github.com/casadi/promo/blob/master/README.md
